package.json

```diff
@@ -216,5 +217,15 @@
       "cloudteam-td-agent-setup",
       "dial-agent"
     ]
+  },
+  "lint-staged": {
+    "*.js": [
```

Comment:
This will need to run against the same file extensions that we current lint i.e. *.{js,jsx,json} at the moment this only runs against .js files.

We also do not current use the auto-fix option so this should be removed.

Something like
```
"lint-staged": {
    "*.{js,jsx,json}": "eslint"
  }
```

Reply:
Done